projects/packages/videopress/src/class-access-control.php

```diff
@@ -0,0 +1,399 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
Nit-pick, but you can actually remove this phpcs exception at this point, your file name is valid. 👍

```suggestion
<?php
```